troposphere/static/js/components/images/detail/ViewImageDetails.react.js

```diff
@@ -23,32 +22,35 @@ export default React.createClass({
         var profile = stores.ProfileStore.get(),
             image = this.props.image;
 
-        if (profile.id && profile.get("username") === image.get("created_by").username || profile.get("is_staff")) {
+        if (profile.id
+            && profile.get("username")
+            === image.get("created_by").username
+            || profile.get("is_staff")) {
             return (
-            <div className="edit-link-row clearfix">
-                <a className="pull-right" onClick={this.props.onEditImageDetails}><span className="glyphicon glyphicon-pencil"></span> Edit details</a>
-            </div>
+                <div>
+                    <a
+                        onClick={this.props.onEditImageDetails}
+                    >
+                        <span className="glyphicon glyphicon-pencil"/>
+                           {" Edit details"}
+                        </a>
+                </div>
             )
         }
     },
 
     render: function() {
-        let tagsView = (
-        <TagsView image={this.props.image} tags={this.props.tags} />
-        );
-
         return (
-        <div>
-            <div>
-                <NameView image={this.props.image} />
-                <CreatedView image={this.props.image} />
-                <RemovedView image={this.props.image} />
-                <AuthorView image={this.props.image} />
-                <DescriptionView image={this.props.image} />
-                {tagsView}
+            <div style={{ marginBottom: "80px", maxWidth: "600px" }}>
```

Comment:
Add this to the top of render, and remove the redundancy below.
`let { image, tags } = this.props`